packages/grafana-ui/src/components/Combobox/MultiCombobox.tsx

```diff
@@ -353,6 +354,7 @@ export const MultiCombobox = <T extends string | number>(props: MultiComboboxPro
                   const itemProps = getItemProps({ item, index });
                   const isSelected = isOptionSelected(item);
                   const id = 'multicombobox-option-' + item.value.toString();
+                  const groupHeaderid = 'multicombobox-option-group-' + item.value.toString();
```

Comment:
@joshhunt did we say to use `combobox-option` and `combobox-option-group` here instead?